charts/scorpio-broker-dist/values.yaml

```diff
@@ -0,0 +1,702 @@
+## namespace for deployment
```

Comment:
We are trying to comply with the helm best practices. Can you please add some short documentation to all values? See https://helm.sh/docs/chart_best_practices/values/#document-valuesyaml
In order to get a nice markdown file for the folder documentation of your chart, check this tool: https://github.com/norwoodj/helm-docs
All other charts use it to generate their documentation(f.e. https://github.com/FIWARE/helm-charts/tree/main/charts/orion) and it would help to get a consistent presentation of the charts.

Reply:
We have incorporated the review comment. Please review again